src/main/java/com/cgm/infolab/controller/ChatService.java

```diff
@@ -33,7 +36,17 @@ public ChatService(UserRepository userRepository,
     }
     public void ChatServiceMetodo(ChatMessage message){
 
-        Timestamp timestamp = new Timestamp(System.currentTimeMillis()); // TODO: rimuovere quando arriverà dal FE
+        // TODO: rimuovere quando arriverà dal FE.
+        // Tutto questo è per simulare il fatto che arriverà una string dal frontend, che andrà
+        // poi convertita in LocalDateTime.
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS");
```

Comment:
Così come c'è una annotation per la serializzazione di `LocalDateTime` a stringa, mi aspetto che ci sia una annotazione (magari la stessa da mettere però sul getter) anche per la deserializzazione da API

Reply:
In realtà no... si creare un progetto giocattolo in cui testare solo questo specifico aspetto 😉. Però va bene anche aspettare che arrivi dal front-end 😊